splunk_eventgen/eventgen_nameko_controller.py

```diff
@@ -461,14 +457,15 @@ def receive_volume(self, data):
     def process_server_status(self, current_time, num_retries=15, delay=0.3):
         current_server_vhosts = self.get_current_server_vhosts()
         server_time = self.server_status['time'] if 'time' in self.server_status else 0
-        server_vhost_len = len(self.server_status) if 'time' not in self.server_status else len(self.server_status)-1
+        server_vhost_len = len(self.server_status) if 'time' not in self.server_status else len(self.server_status) - 1
         if current_server_vhosts:
             for i in range(num_retries):
                 if server_vhost_len != len(current_server_vhosts) or server_time < current_time:
                     time.sleep(delay)
                     current_server_vhosts = self.get_current_server_vhosts()
                     server_time = self.server_status['time'] if 'time' in self.server_status else 0
-                    server_vhost_len = len(self.server_status) if 'time' not in self.server_status else len(self.server_status)-1
+                    server_vhost_len = len(
```

Comment:
weird

Reply:
I think for these kinds of long line code , one option is to break the lines instead of formatting it. I make it more readable. 
```
if 'time' not in self.server_status:
    server_vhost_len = len(self.server_status)
else:
    server_vhost_len = len(self.server_status) - 1
```